docs/diagrams/GraphicalRepresentationClassDiagram.puml

```diff
@@ -1,22 +1,24 @@
 @startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor MODEL_COLOR
+skinparam classBackgroundColor MODEL_COLOR
 
 class ExpenseBook
 class UniqueExpenseList
 class ObservableList
 class Expense
 class PieChartData
-interface ChartData
-class ChartDisplay
+interface ChartDataCollector <<Interface>>
 class GraphicalDisplayWindow
```

Comment:
I think `GraphicalDisplayWindow` (and the arrow from it) should share the color UI_COLOR since it is from the UI component rather than the Model component.

Reply:
Thanks for pointing it out!